### PR TITLE
feat: added home page

### PR DESCRIPTION
**Pull request by kshitijthatte**
- added navbar
- added categories section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.card-img {
+  object-fit: cover;
+}
+
+@media (max-width: 650px) {
+  .categories {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .new-arrivals {
+    grid-template-columns: repeat(1, 1fr);
```
Comment (srishti-maurya) [inline style.css]: you can write it as `grid-template-columns: 1fr;`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  .new-arrivals .card {
+    max-width: 100%;
+    justify-self: stretch;
+  }
+}
+@media (max-width: 400px) {
+  .categories {
+    grid-template-columns: repeat(1, 1fr);
```
Comment (srishti-maurya) [inline style.css]: you can write it as `grid-template-columns: 1fr;`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  width: 100%;
+  border-radius: 4px;
+}
+
+.new-arrivals .c-badge-right {
+  z-index: 2;
+}
+
+.nav-fixed {
+  z-index: 9999;
```
Comment (srishti-maurya) [inline style.css]: Suggestion: it's not necessary to use this high value for `z-index`, `z-index: 5 or 10` may also work i guess

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by msdeshmukh009 — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            <div class="card-title">Cookies</div>
+          </div>
+        </a>
+        <a class="card card-overlay" href="/cakes.html">
+          <img
+            class="card-img category-img"
+            src="https://source.unsplash.com/qZ6uvJHLHFc"
+          />
+          <div class="overlay-content">
+            <div class="card-title">Doughnuts</div>
```
Comment (MarkVed17) [inline index.html:61]: Donuts*?

**Review by MarkVed17 — APPROVED** [review]

**Comment by preethasri** [comment]
Looks good 👍

File: style.css (hunk 1 above)
Comment (kshitijthatte) [reply style.css]: Yeah right, thanks!

File: style.css (hunk 3 above)
Comment (kshitijthatte) [reply style.css]: Was just trying to be too sure that it stays on the top 😅, will change it.

File: index.html (hunk 4 above)
Comment (kshitijthatte) [reply index.html:61]: Both are correct, Donut is actually a simplified spelling of Doughnut.

**Pull request merged**